cou/steps/backup.py

```diff
@@ -24,7 +25,7 @@
 logger = logging.getLogger(__name__)
 
 
-async def backup(model_name: Optional[str] = None) -> str:
+async def backup(model_name: Optional[str] = None) -> Path:
```

Comment:
change `:rtype: str` to  `:rtype: Path`  in docstring

Reply:
ok